src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -988,6 +1114,41 @@ public override FFIFunctionPointer GetFunctionPointer(string className, string n
             return null;
         }
 
+        /// <summary>
+        /// Creates CLRModuleTypes for all types in this module.
+        /// </summary>
+        /// <param name="isReflectionContext">When true the behavior for creating AST Nodes for the module types is using specific ReflectionOnly methods</param>
+        public override void ScanModule(bool isReflectionContext)
+        {
+            CLRModuleType.IsReflectionContext = isReflectionContext;
+
+            Type[] types = isReflectionContext ? 
+                GetReflectionTypes(string.Empty) : 
+                GetTypes("");
+
+            foreach (var type in types)
+            {
+                //For now there is no support for generic type.
+                if (type.IsGenericType || 
+                    !type.IsPublic || 
+                    CLRModuleType.SupressesImport(type)) continue;
```

Comment:
should we also log these?

Reply:
Not sure that would be useful as this is just filtering out everything that arent nodes? i would say its expected that we arent creating docs for them.